### DISCUSSION TITLE
Selftime for a many services(apis) are coming as negative number in NodeGraph

### DISCUSSION THREAD

**Discussion by sumitkgaur** [open-telemetry/community#1763] [Community Discussions]
It is observed that Node Graph has many services (API) with negative Selftime as per the image attached. how that is explained if that is right ?

![Screen Shot 2023-11-01 at 3 40 04 pm](https://github.com/open-telemetry/community/assets/5743600/62f5e77e-e140-43cf-a452-b9dde1fe5736)

**Comment by tigrannajaryan** [comment]
How is this related to OpenTelemetry?

**Comment by sumitkgaur** [comment]
Stack consist opentelemetry auto instrumentation to Otel collector to tempo
visualisation.

Let me know if question is still for wrong group.  Otel data is not coming
correct and getting visualised in negative numbers

**Reply by trask** [reply]
hi @sumitkgaur, there are several components involved here, I'd recommend trying to narrow down where the negative numbers are coming from (e.g. a particular instrumentation, SDK, visualization component), and then opening a issue/discussion in the repository for that component

**Reply by sumitkgaur** [reply]
understnad, as we are seeing this problem only in selftime calculations and it is hard to trace to from consolidate view to each and every transaction. I would give it to clock synchronization.
